src/dropdown.ts

```diff
@@ -2047,19 +2187,90 @@ export default class Dropdown extends LitElement implements FormControl {
 
   #onOptionsDisabledChange(event: Event) {
     if (
+      this.multiple &&
       event.target instanceof DropdownOption &&
-      event.target.disabled &&
-      event.target.selected
```

Comment:
`event.target.selected` is now guarded against below so that the `event.target.privateActive` condition also below gets hit.